src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -714,6 +715,7 @@ export class InlineChatController implements IEditorContribution {
 
 			const canContinue = this._strategy.checkChanges(response);
 			if (!canContinue) {
+				this._chatAccessibilityService.acceptResponse();
```

Comment:
should we say something here? what happens for a sighted user in this case?

Reply:
The refactor preview flow is started from here

Reply:
Ok, so I think we don't need to say anything then?